src/main/java/com/minecolonies/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -121,47 +125,47 @@ protected AbstractEntityAIBasic(@NotNull final J job)
     {
         super(job);
         super.registerTargets(
-                /**
+                /*
                  * Init safety checks and transition to IDLE
                  */
-                new AITarget(INIT, this::initSafetyChecks),
-                /**
+          new AITarget(INIT, this::initSafetyChecks),
+                /*
                  * Update chestbelt and nametag
                  * Will be executed every time
                  * and does not stop execution
                  */
-                new AITarget(this::updateVisualState),
-                /**
+          new AITarget(this::updateVisualState),
+                /*
                  * If waitingForSomething returns true
                  * stop execution to wait for it.
                  * this keeps the current state
                  * (returning null would not stop execution)
                  */
-                new AITarget(this::waitingForSomething, this::getState),
-                /**
+          new AITarget(this::waitingForSomething, this::getState),
+                /*
                  * Check if any items are needed.
                  * If yes, transition to NEEDS_ITEM.
                  * and wait for new items.
                  */
-                new AITarget(() -> !itemsCurrentlyNeeded.isEmpty(), this::waitForNeededItems),
-                /**
+          new AITarget(() -> !itemsCurrentlyNeeded.isEmpty(), this::waitForNeededItems),
+                /*
                  * Wait for different tools.
                  */
-                new AITarget(() -> this.needsShovel, this::waitForShovel),
-                new AITarget(() -> this.needsAxe, this::waitForAxe),
-                new AITarget(() -> this.needsHoe, this::waitForHoe),
-                new AITarget(() -> this.needsPickaxe, this::waitForPickaxe),
-                /**
+          new AITarget(() -> this.needsShovel, this::waitForShovel),
+          new AITarget(() -> this.needsAxe, this::waitForAxe),
+          new AITarget(() -> this.needsHoe, this::waitForHoe),
+          new AITarget(() -> this.needsPickaxe, this::waitForPickaxe),
+                /*
                  * Dumps inventory as long as needs be.
                  * If inventory is dumped, execution continues
                  * to resolve state.
                  */
-                new AITarget(INVENTORY_FULL, this::dumpInventory),
-                /**
+          new AITarget(INVENTORY_FULL, this::dumpInventory),
+                /*
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Block comment has incorrect indentation level 16, expected is 10, indentation should be the same level as line 167. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck)